### PR TITLE
Split the MiSeq graph into v2 and v3, added series names and colors for HiSeqs/MiSeqs in chemver

### PR DESCRIPTION
**Pull request by sofiahaglund**
As a first step to adding the Nano runs, I split the graph into v2 and v3 for the MiSeqs (I guess Nano is always v2, but not all v2 are Nanos)...

I also added the series names for the Miseqs, HiSeqs and changed the colors. It's on stage!

PS. I gave up on fixing the trailing spaces, my Atom settings aren't updating or something 🤷‍♀️DS.

### CODE REVIEW

**Comment by aanil** [comment]
Is this up on stage?

File: run_dir/static/js/fc_trend_plot.js [hunk 1]
```diff
@@ -281,26 +297,42 @@ function color_by_chemistry(chem){
...
-   return current_color_schemes[6](id).hex();  
+   return current_color_schemes[6](id).hex();
 }
-        
+
 function color_by_sp(chem){
   var id = Math.round(window.current_chemistries_list.indexOf(chem));
    return current_color_schemes[7](id).hex();  
 }
+function color_by_v2(chem){
```
Comment (aanil) [inline run_dir/static/js/fc_trend_plot.js]: Might we not have just one of there 'color_by' functions as they seem to be doing the same thing? You could have a dictionary in it mapping id to the hardcoded number.

Comment (sofiahaglund) [reply run_dir/static/js/fc_trend_plot.js]: hmm I'll have to look into it 👍

Comment (sofiahaglund) [reply run_dir/static/js/fc_trend_plot.js]: done!

**Comment by aanil** [comment]
Yes, but it always returns the same value (-1) because the `series_name` values we use do not coincide with what is in `window.current_chemistries_list`. So I'm not sure why we use it? We should change one or the other so that the values are consistent and I'm not sure if its worth the effort because it seems to work for now.

**Pull request merged**